### PR TITLE
mirage: pass static ip config to `make`

### PR DESCRIPTION
**Pull request by mor1**
pass in IP configuration as env vars

### CODE REVIEW

**Comment by mor1** [comment]
@samoht for context: changes to support @avsm suggestion to pass in static IP parameters for deployment unikernel from environment

File: .travis-mirage.sh [hunk 1]
```diff
@@ -11,13 +11,15 @@ export OPAMYES=1
...
+DEPLOY=$DEPLOY MODE=$MIRAGE_BACKEND NET=$MIRAGE_NET \
+      ADDR=$MIRAGE_ADDR MASK=$MIRAGE_MASK GWS=$MIRAGE_GWS \
+      make configure
 make build
 
 ## stash deployment build if specified
 if [ "$DEPLOY" = "1" \
                -a "$TRAVIS_PULL_REQUEST" = "false" \
-               -a -n "$SECRET_default_0" ]; then
+               -a -n "$XSECRET_default_0" ]; then
```
Comment (samoht) [inline .travis-mirage.sh]: Is the extra `X` intensional?

Comment (avsm) [inline .travis-mirage.sh]: XSECRET is correct; https://github.com/avsm/travis-senv/blob/master/main.ml#L33

Comment (samoht) [inline .travis-mirage.sh]: ok merging then

**Pull request merged**